build_defs/run_in_repo_template.sh

```diff
@@ -1,5 +1,3 @@
 #!/bin/bash
 set -e
-# cd to repo root first before running command.
-cd "{repo_root}"
 {cmd}
```

Comment:
It doesn't look like this file is needed at all anymore. Can you delete it, and instead of running `ctx.actions.expand_template` just run the command instead (using `ctx.actions.run` or `ctx.actions.run_shell`).

Reply:
Seems like arcs_ts_test still wants a shell script returned in DefaultInfo provider (e.g. executable=script_file), so may need to keep it?